1/contrib/s2i/run

```diff
@@ -9,6 +9,29 @@
 source /usr/local/bin/jenkins-common.sh
 source /usr/local/bin/kube-slave-common.sh
 
+#display the available i386 and x86_64 java's to confirm their presence and version
+alternatives --display java | grep family | grep i386
+alternatives --display java | grep family | grep x86_64
```

Comment:
is this just for debugging purposes?  doesn't seem like output we'd likely want in the final image.